.github/ISSUE_TEMPLATE/6-bug-report-other.yaml

```diff
@@ -0,0 +1,122 @@
+name: "🐛 Report a Bug With Another Package"
+description: "Report a bug you encountered with another one of our packages (parser, util, scope-manager, etc)"
+title: "Bug: <short description of the issue>"
+labels:
+  - bug
+  - triage
+body:
+  - type: checkboxes
+    id: sanity-checks
+    attributes:
+      label: Before You File a Bug Report Please Confirm You Have Done The Following...
+      description: If any of these required steps are not taken, we may not be able to review your issue. Help us to help you!
+      options:
+        - label: I have tried restarting my IDE and the issue persists.
+          required: true
+        - label: I have updated to the latest version of the packages.
+          required: true
+        - label: I have [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Abug) and found none that matched my issue.
+          required: true
+        - label: I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.
+          required: true
+  - type: markdown
+    id: complexity-note
+    attributes:
+      value: |
+        ## Note For Complex Issues
+        If your issue requires some complex reproduction - such as multiple files or a specific folder structure - please do not use this template.
+        Instead, select the ["Report a Complex Bug With a Reproduction Repository" template](https://github.com/typescript-eslint/typescript-eslint/issues/new?template=7-bug-report-complex.yaml).
+        ---
```

Comment:
Is this intentional? This makes everything an h2 heading.

Reply:
This was intentional yeah - because this markdown header uses a H2, GH renders a line under the heading. None of the other form field labels have lines (I think they're H2 as well).

It felt more aesthetically pleasing to "close off" this section with a line as well.

![Screenshot_20220506-082053.png](https://user-images.githubusercontent.com/7462525/167163087-470160c0-ad58-4e7b-83ce-9349dc491069.png)

Reply:
It looks kind of "noisy" though. (My opinion)

Have you adding a line break above this? (Disclaimer: I haven't)

Reply:
Oooohhh I see what you mean niw. Sorry I misunderstood.

Yes the paragraph should not be a H2.
Thr line was supposed to be separate.